### PR TITLE
Fix broken links to RN docs

### LINKED ISSUE #857: Broken link to RN tutorial docs
Current `readme` has a [broken link](https://reactnative.dev/docs/tutorial.html) to the React Native docs which could be confusing for a lot of new folks. 

image

### PR DESCRIPTION
**Pull request by kartikcho**
Resolves #857 

Removed `.html` prefix from the link.

Edit: Updated all the other links to point to the correct sources too so it doesn't 404 anymore.

### CODE REVIEW

**Comment by slorber** [comment]
@kartikcho we'll fix this at the RN website level, so it's not needed to fix links here

**Comment by Simek** [comment]
@slorber I think it would be nice to fix the links, regardless of the redirect on the RN site. 🙂

**Comment by kartikcho** [comment]
@slorber like @Simek said, I think it would be a good to have.

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ By default you create a [bare-workflow React](https://docs.expo.io/bare/explorin
 
 ## Usage with Expo Client App
 
-Expo Client enables you to work with all of the [Components and APIs](https://facebook.github.io/react-native/docs/getting-started.html) in `react-native`, as well as the [JavaScript APIs](https://docs.expo.io/versions/latest/sdk/index.html) that the are bundled with the Expo App.
+Expo Client enables you to work with all of the [Components and APIs](https://facebook.github.io/react-native/docs/getting-started) in `react-native`, as well as the [JavaScript APIs](https://docs.expo.io/versions/latest/sdk/index) that the are bundled with the Expo App.
```
Comment (Simek) [inline README.md:72]: The correct link to latest Expo docs is https://docs.expo.io/versions/latest/.

Before the change (with `.html) link cause a redirect to the correct page, now it leads to an error page:
Screenshot 2020-11-02 163900

Comment (kartikcho) [reply README.md:72]: Ah, thanks! Should I revert it or update it to the newer link?

Comment (Simek) [reply README.md:72]: I'm not the CRNA maintainer but I would recommend updating the Expo link to the new format. 🙂

**Comment by slorber** [comment]
make sense :) 

maybe we can find all all RN sites incoming links using the .html extension through analytics and fix them all :)

**Comment by kartikcho** [comment]
I've updated all the links in the readme, are their any other docs that need fixing in this repo @slorber ?

**Comment by slorber** [comment]
I don't know, just know that you'd rather refactor the xyz.html links :)

**Review by EvanBacon — APPROVED** [review]
LGTM

**Pull request merged**